client/app/states/services/list/list.state.js

```diff
@@ -238,15 +238,15 @@
     }
 
     function matchesFilter(item, filter) {
-      if ('name' === filter.id) {
+      if (filter.id === 'name') {
```

Comment:
Didn't know something called `Yoda conditions` existed
https://en.wikipedia.org/wiki/Yoda_conditions
Good to know :)

But looks like the constant portion should be on the left side?... and you flipped it?
(A yoda condition places the constant portion of the expression on the left side of the conditional statement)